GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -594,12 +572,6 @@ - (void)testSignOut {
   XCTAssertNil([_keychainHandler loadAuthState]);
 }
 
-- (void)testNotHandleWrongScheme {
```

Comment:
Has this test case been moved elsewhere?

Reply:
No, this test case is deleted because it is testing the method `resumeExternalUserAgentFlowWithURL:` in `GIDAuthorizationFlowProcessor`. We have tests in `GIDAuthorizationFlowProcessorTest` to handle this case.
And the next test case `testNotHandleWrongPath` handles the path checking.